src/planning/plan/tests/replan-queue-test.ts

```diff
@@ -43,6 +45,7 @@ async function init(options?) {
   `);
   const runtime = new Runtime({loader, context});
   const arc = runtime.newArc('test');
+  VolatileStorageDriverProvider.register(arc);
```

Comment:
this is done as part of the Arc ctor already: https://github.com/PolymerLabs/arcs/blob/master/src/runtime/arc.ts#L131